### PR TITLE
Created a dockerfile that can be used to run the integration test

### PR DESCRIPTION
**Pull request by roxaneletourneau**
1- Start the sandbox as described here: https://gitlab.com/tezos/teci/-/blob/main/docker/example_test_suite/README.md
2- In the root folder of taquito run the following command:
`docker build -t taquito .`
`docker run --env RUN_MONDAYNET_WITH_SECRET_KEY=true --env SECRET_KEY=edsk3gUfUPyBSfrS9CCgmCiQsTCHGkviBDusMxDJstFtojtc1zcpsh --network host --env TEZOS_RPC_MONDAYNET=http://0.0.0.0:8732/ --env POLLING_INTERVAL_MILLISECONDS=100 --env RPC_CACHE_MILLISECONDS=0 --env TEZOS_MONDAYNET_BAKER=tz1gjaF81ZRRvdzjobyfVNsAeSC6PScjfQwN taquito`

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/originate-known-contracts.ts [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
   let keyPkh: string = "";
   let keyInitialBalance: BigNumber = new BigNumber(0);
 
   (async () => {
     await setup(true);
+    fs.writeFile(`known-contracts-${protocol.substring(0,9)}.ts`, '', (err: any) => {
+      if (err) {
+        console.error(err);
+      }
+    });
```
Comment (zainen) [inline integration-tests/originate-known-contracts.ts:25]: each test we would like to overwrite the last tests file? 
this doesnt throw an error if the file already exists right?

**Review by zainen — APPROVED** [review]
LGTM just one question about fs.write file

File: integration-tests/originate-known-contracts.ts [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-import { knownContract } from '../example/data/knownContract';
-import { knownBigMapContract } from '../example/data/knownBigMapContract';
 import { singleSaplingStateContractJProtocol } from './data/single_sapling_state_contract_jakarta_michelson';
 import { fa2ForTokenMetadataView } from './data/fa2-for-token-metadata-view';
 import { char2Bytes } from '@taquito/utils';
 import BigNumber from 'bignumber.js';
 import { codeViewsTopLevel } from './data/contract_views_top_level';
+import { knownBigMapContract } from './data/knownBigMapContract';
+import { knownContract } from './data/knownContract';
+const fs = require('fs');
```
Comment (dsawali) [inline integration-tests/originate-known-contracts.ts:10]: are we not able to use import for fs?

**Review by hui-an-yang — APPROVED** [review]

File: integration-tests/originate-known-contracts.ts (hunk 1 above)
Comment (roxaneletourneau) [reply integration-tests/originate-known-contracts.ts:25]: Yes, it overrides the latest file, or it creates one if it doesn't exist.

File: integration-tests/originate-known-contracts.ts (hunk 2 above)
Comment (roxaneletourneau) [reply integration-tests/originate-known-contracts.ts:10]: Yes import would would work

Comment (dsawali) [reply integration-tests/originate-known-contracts.ts:10]: I think we should import for consistency

Comment (roxaneletourneau) [reply integration-tests/originate-known-contracts.ts]: Yes, thanks I made the change

**Pull request merged**